src/views/Benchmark/index.jsx

```diff
@@ -35,6 +39,26 @@ class Benchmark extends Component {
     this.mounted = false;
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  handleTooltipClick(e, items) {
+    const globalChartObj = Chart;
+    // check if the click happened on empty chart space or data point
+    if (items.length > 0) {
+      if (globalChartObj.lastClickedDataPoint === items[0]) {
+        // val to null is used to close the open tooltip
+        globalChartObj.lastClickedDataPoint = null;
+      } else {
+        [globalChartObj.lastClickedDataPoint] = items;
+      }
+    } else {
+      globalChartObj.lastClickedDataPoint = null;
```

Comment:
This whole block of conditionals could be made more concise (cut down to fewer lines of code).